pkg/install/install.go

```diff
@@ -99,7 +99,8 @@ func RunUninstall(cmd *cobra.Command, args []string) {
 // RunStatus runs a CLI command
 func RunStatus(cmd *cobra.Command, args []string) {
 	log := util.Logger()
-	version.RunVersion(cmd, args)
+
+	system.RunSystemVersionsStatus(cmd, args)
```

Comment:
Few lines below we are calling system.RunStatus which already loads the system and a lot of the current status, so why aren't we just adding this functionality over there? I think in general we should show both the CLI version information clearly, and also the current system installation versions clearly, each one in its own  section. But as it is now I think it will still be confusing.

Reply:
2 reasons for that:
1. I wanted to preserve the structure of the output. 
2. RunUninstall is not calling system.RunStatus and I wanted to fix the images there also.

Reply:
ok